tests/basic_tests.lua

```diff
@@ -433,7 +455,7 @@ function df_tests._update_single_row()
     ['Col C']=4
   }
 	a:_update_single_row(1, new)
-  tester:assertTableEq(a:_extract_row(1), new)
+  tester:assertTableEq(a:get_row(1), new)
 end
 
```

Comment:
Missing one-valued ```load_table``` test. I tested this code :
```lua
function df_tests.load_table_test()
  local a = Dataframe()
  a:load_table{data={['first_column']=3,['second_column']={10,11,12}}}
  
end
```

Which never pass according to the first above comments